src/cta-button/editor.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * #.# Editor Styles
+ *
+ * CSS for just Backend enqueued after style.scss
+ * which makes it higher in priority.
+ */
+
+.bulma-blocks-editor-link-card {
+
+  padding: 0.875rem;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #000;
+  background: #f3f3f3;
+  margin-bottom: 1.75rem;
+
+  .content {
+    width: 100%;
+    max-width: 30rem;
+
+    span {
+      display: flex;
+      margin-bottom: 1rem;
+    }
+
+    &:first-child {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+    .field{
+      .control{
+        input{
+          background-color:#fff;
+        }
+      }
+    }
+  }
+
+  &__container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: #f3f3f3;
+    color: black;
```

Comment:
perhaps this could be changed to ```#000000``` to remain consistent with the rest of the color hexcodes